fontc/src/lib.rs

```diff
@@ -678,6 +678,47 @@ mod tests {
         assert_eq!(expected, completed);
     }
 
+    /// A convenience type for the common case of "just compile this source with
+    /// default arguments".
+    ///
+    /// When you need more control, just the `compile` fn/`TestCompile` type
+    /// directly
+    struct SimpleCompile {
```

Comment:
I like the elimination of boilerplate but I'm not sure why you need a new type, just improve TestCompile?

Reply:
Filed as https://github.com/googlefonts/fontc/issues/594 to unblock merging